spec/command_tests/table_spec.ts

```diff
@@ -0,0 +1,109 @@
+import {initServer} from './helpers';
+import {Session} from '../mock-server/interfaces';
+import * as path from 'path';
+
+export interface Testcase {
+  skip?: boolean;
+  args?: any[];
+  result?: any;
+  session?: Session;
+  params?: {[name: string]: any};
+}
+
+function check(actual: any, expected: any) {
```

Comment:
`actual` may have some properties not in `expected`.  For instance, I'm not checking the `sessionId`, which is on the `params` object.  I don't run any checks against the `session` object, but if I did this would be helpful there too (since I probably only want to check one or two fields in the `session`).  I could use `toEqual` for the `result` verification though, and I could also use `toEqual` on all properties of `expected`, rather than doing a recursive check.